drip/admin.py

```diff
@@ -2,6 +2,8 @@
 
 from django import forms
 from django.contrib import admin
+from django.db.models import UUIDField
```

Comment:
Are you using this import? I think you only check the type of ID with `get_internal_type` function, if that is the case please remove it

Reply:
Corrected this.